reconcile/openshift_saas_deploy_wrapper.py

```diff
@@ -29,6 +35,11 @@ def run(dry_run, thread_pool_size=10, io_dir='throughput/'):
             thread_pool_size,
             len(saas_file_owners_diffs))
 
-    threaded.run(osd_run_wrapper, saas_file_owners_diffs, thread_pool_size,
-                 dry_run=dry_run,
-                 available_thread_pool_size=available_thread_pool_size)
+    exit_codes = threaded.run(
+        osd_run_wrapper, saas_file_owners_diffs, thread_pool_size,
+        dry_run=dry_run,
+        available_thread_pool_size=available_thread_pool_size
+    )
+
+    if [ec for ec in exit_codes if ec > 0]:
```

Comment:
`if [ec for ec in exit_codes if ec]:`, so we get any non-0, including negatives.

Reply:
is that not good?

Reply:
done!